src/Day.jsx

```diff
@@ -7,6 +7,14 @@ let Day = React.createClass({
 
   propTypes: TimeGrid.propTypes,
 
+  getDefaultProps() {
+    return {
+      now: new Date(),
+      type: 'gutter',
+      timeslots: 2
+    }
```

Comment:
To me its a big confusing to define defaults for props that the component doesn't explicitly use. Its makes it a bit harder to understand which value will be used ultimately and harder to change the default in the right spot.

Can we remove them for these components?